elements/rh-tabs/demo/nested.html

```diff
@@ -28,6 +28,19 @@
   rh-tab-panel {
     padding: 0;
   }
+
+  a {
+    color: var(--rh-color-interactive-primary-default);
+    &:hover { color: var(--rh-color-interactive-primary-hover); }
+    &:focus-within { color: var(--rh-color-interactive-primary-focus); }
+    &:active { color: var(--rh-color-interactive-primary-active); }
+    &:visited {
+      color: var(--rh-color-interactive-primary-visited);
+      &:hover { color: var(--rh-color-interactive-primary-visited-hover); }
+      &:focus-within { color: var(--rh-color-interactive-primary-visited-focus); }
+      &:active { color: var(--rh-color-interactive-primary-visited-active); }
+    }
+  }
```

Comment:
@bennypowers instead of adding these to the demo, could we add [base layer styles](https://github.com/RedHat-UX/red-hat-design-system/blob/c913b8fb9bb8f167d8704ceb67f383ceacb34672/docs/styles/styles.css#L26-L75) via the changes in https://github.com/RedHat-UX/red-hat-design-system/pull/1862 to [docs/styles/demo/styles.css](https://github.com/RedHat-UX/red-hat-design-system/pull/1862/files#diff-611152b35fc6b837bacde981148e7e8c227cb2f10f6ca037074df6ee66651c7a) for demos.  To me it only makes sense to remove these from the demo css as it really has nothing to do with the tabs demos themselves.

Reply:
i dont' want there to be any dependencies on the demos except the global css at maximum

a user should be able to copy paste the entire demo into a fresh page and get the same results or near-to